js/app/controllers/calendarlistcontroller.js

```diff
@@ -148,9 +148,9 @@ app.controller('CalendarListController', ['$scope', '$rootScope', '$window', 'Ha
 			MailerService.sendMail(item.email, item.publicSharingURL, item.calendar.displayname).then(function (response) {
 				if (response.status === 200) {
 					item.email = '';
-					OC.Notification.showTemporary(t('calendar', 'Email has been sent.'));
+					OC.Notification.showTemporary(t('calendar', 'Email sent.'));
 				} else {
-					OC.Notification.showTemporary(t('calendar', 'There was an issue while sending your email.'));
```

Comment:
Please change it back to `There was an issue while sending your email.` and don't come up with new phrases for this very string please. There is nothing wrong about `There was an issue while sending your email.`

Reply:
While true, in the greater scheme of things, there are 83 "Could not"-errors, and only 8 "There was"-errors. Two of which are the ones dealt with here.
My thoughts on the matter is "could not" is shorter, and "There was an issue" is ambiguous as to whether it is a failure or not. One could argue the email got sent, and that the issue was some headers were omitted or anything else that amounts to an issue.